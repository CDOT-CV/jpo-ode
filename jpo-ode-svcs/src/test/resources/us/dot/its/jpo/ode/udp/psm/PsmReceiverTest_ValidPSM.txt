011d0000201a0000021bd86891de75f84da101c13f042e2214141fff00022c2000270000000163b2cc7986010000