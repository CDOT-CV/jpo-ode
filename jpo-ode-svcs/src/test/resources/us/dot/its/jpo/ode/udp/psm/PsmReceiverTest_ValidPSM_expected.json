{
  "metadata" : {
    "recordType" : "psmTx",
    "securityResultCode" : "success",
    "payloadType" : "us.dot.its.jpo.ode.model.OdeAsn1Payload",
    "serialId" : {
      "streamId" : "33efc65d-f138-4c77-972c-1584d936e63e",
      "bundleSize" : 1,
      "bundleId" : 0,
      "recordId" : 0,
      "serialNumber" : 0
    },
    "odeReceivedAt" : "2024-11-26T23:53:21.120Z",
    "schemaVersion" : 8,
    "maxDurationTime" : 0,
    "recordGeneratedBy" : "UNKNOWN",
    "sanitized" : false,
    "asn1" : "00201A0000021BD86891DE75F84DA101C13F042E2214141FFF00022C2000270000000163B2CC7986010000",
    "psmSource" : "RSU",
    "originIp" : "127.0.0.1"
  },
  "payload" : {
    "dataType" : "us.dot.its.jpo.ode.model.OdeHexByteArray",
    "data" : {
      "bytes" : "00201A0000021BD86891DE75F84DA101C13F042E2214141FFF00022C2000270000000163B2CC7986010000"
    }
  }
}